spec/features/user_creates_bot_spec.rb

```diff
@@ -32,7 +32,7 @@
     select("Male", from: "bot[gender]")
     select("Green", from: "bot[eye_color]")
     select("Black", from: "bot[hair_color]")
-    fill_in("Age", with: 34)
+    select("26", from: 'bot[age]')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.